Apps/WebClient/src/ClientApp/src/views/login.vue

```diff
@@ -82,7 +82,7 @@ export default class LoginView extends Vue {
 </script>
 
 <template>
-    <div class="container my-5" align="center">
```

Comment:
This looks like is changing things outside the scope of the PR

Reply:
is this really a big deal?

Reply:
it could be, but I am not sure since it could be changing the display of the div changing the UI. Moreover is not aligned with the stated purpose of the PR. Can you explain why you changed it?

Reply:
styling attributes like "align, width, height" are obsolete